sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -894,6 +895,87 @@ class DataSourceV2SQLSuite
     }
   }
 
+  test("create/replace/alter table - reserved properties") {
+    import TableCatalog._
+    withSQLConf((SQLConf.LEGACY_PROPERTY_NON_RESERVED.key, "false")) {
+      RESERVED_PROPERTIES.asScala.filterNot(_ == PROP_COMMENT).foreach { key =>
```

Comment:
shall we just remove PROP_COMMENT from RESERVED_PROPERTIES?

Reply:
I suggest we revisit it later, cuz' now we treat `comment` as a special field like `location` and `provider` in many places. e.g. CTAS/RTAS statement, DESC commands. Although this change is small, it might change a lot of relative logic.

FYI, We can do this when we migrate CT/CTAS, etc to new the new plan resolver.

Reply:
Sounds good